modules/controller/rpc-controller.js

```diff
@@ -264,8 +264,17 @@ class RpcController {
                         },
                     },
                 );
-
             } catch (e) {
+                Models.handler_ids.update(
```

Comment:
handler_id can be null at this point, we should handle that and return error to the user in response

Reply:
Resolved with [this commit](https://github.com/OriginTrail/ot-node/pull/1738/commits/6060684f171448393248ba96a1bf5601e43da390)